src/js/browser-plugin.js

```diff
@@ -172,6 +172,7 @@ const publishTweetCallback = (clickEvent, targetButton) => {
     return true;
   }
 
+  log2Server('publish tweet', urls, `Tweet id: ${tweetText}`, ' click on "publish anyway"');
```

Comment:
Careful, on the third parameter the values name is wrong: "Tweet id: Tweet Text".